examples/public/layers/change-feature-columns-gmaps.html

```diff
@@ -104,12 +104,12 @@ <h1>Change the feature columns</h1>
 
     function setMoreData() {
       layer.setFeatureClickColumns(['name', 'pop_max', 'pop_min']);
-      document.getElementById('content').innerHTML = '';
+      document.getElementById('info').innerHTML = '';
```

Comment:
Was this broken because of a wrong id?

Reply:
Yeah, but nothing serious, that's only used to "clear" a text. I noticed when checking all examples.

Reply:
Good boy scout 😄 !!